doc/hardware/porting/board_porting.rst

```diff
@@ -157,19 +157,19 @@ not always used.
    | Board                                      | Identifier            | SoC         | SoC Series    | SoC family    | CPU core       | Architecture |
    +============================================+=======================+=============+===============+===============+================+==============+
-   | :ref:`nrf52dk <nrf52dk_nrf52832>`          | /nrf52832             | nRF52832    | nRF52         | Nordic nRF    | Arm Cortex-M4  | Arm          |
+   | :ref:`nrf52dk <nrf52dk_nrf52832>`          | nrf52832              | nRF52832    | nRF52         | Nordic nRF    | Arm Cortex-M4  | Arm          |
    +--------------------------------------------+-----------------------+-------------+---------------+---------------+----------------+--------------+
-   | :ref:`frdm_k64f <frdm_k64f>`               | /mk64f12              | MK64F12     | Kinetis K6x   | NXP Kinetis   | Arm Cortex-M4  | Arm          |
+   | :ref:`frdm_k64f <frdm_k64f>`               | mk64f12               | MK64F12     | Kinetis K6x   | NXP Kinetis   | Arm Cortex-M4  | Arm          |
    +--------------------------------------------+-----------------------+-------------+---------------+---------------+----------------+--------------+
-   | :ref:`rv32m1_vega <rv32m1_vega>`           | /openisa_rv32m1/ri5cy | RV32M1      | (Not used)    | (Not used)    | RI5CY          | RISC-V       |
+   | :ref:`rv32m1_vega <rv32m1_vega>`           | openisa_rv32m1/ri5cy  | RV32M1      | (Not used)    | (Not used)    | RI5CY          | RISC-V       |
    +--------------------------------------------+-----------------------+-------------+---------------+---------------+----------------+--------------+
-   | :ref:`nrf5340dk <nrf5340dk_nrf5340>`       | /nrf5340/cpuapp       | nRF5340     | nRF53         | Nordic nRF    | Arm Cortex-M33 | Arm          |
+   | :ref:`nrf5340dk <nrf5340dk_nrf5340>`       | nrf5340/cpuapp        | nRF5340     | nRF53         | Nordic nRF    | Arm Cortex-M33 | Arm          |
    |                                            +-----------------------+-------------+---------------+---------------+----------------+--------------+
-   |                                            | /nrf5340/cpunet       | nRF5340     | nRF53         | Nordic nRF    | Arm Cortex-M33 | Arm          |
+   |                                            | nrf5340/cpunet        | nRF5340     | nRF53         | Nordic nRF    | Arm Cortex-M33 | Arm          |
    +--------------------------------------------+-----------------------+-------------+---------------+---------------+----------------+--------------+
-   | :ref:`mimx8mp_evk <imx8mp_evk>`            | /mimx8m/a53           | i.MX8M Plus | i.MXM8M A53   | NXP i.MX      | Arm Cortex-A53 | Arm64        |
+   | :ref:`mimx8mp_evk <imx8mp_evk>`            | mimx8m/a53            | i.MX8M Plus | i.MX8M        | NXP i.MX      | Arm Cortex-A53 | Arm64        |
```

Comment:
Is `mimx8m` the correct identifier?
In boards/ we have `mimx8ml8`.
See:
A53 -  https://github.com/zephyrproject-rtos/zephyr/blob/main/boards/nxp/imx8mp_evk/imx8mp_evk_mimx8ml8_a53.yaml#L7
M7 - https://github.com/zephyrproject-rtos/zephyr/blob/main/boards/nxp/imx8mp_evk/imx8mp_evk_mimx8ml8_m7.yaml#L7

Reply:
Done